FreeRTOS/FreeRTOS
# Update RISC-V Qemu Virt GCC Readme + Makefile

Description
-----------
Update the RISC-V-Qemu-virt_GCC Readme and Makefile to contain the latest successful demo run information.

Test Steps
-----------
Ran the demo as part of testing https://github.com/FreeRTOS/FreeRTOS/pull/864.

Ran the 'blinky' demo with the vector table enabled. Saw the following output...
```
Hello FreeRTOS!
0: Tx: Transfer1
0: Rx: Blink1
0: Tx: Transfer2
0: Rx: Blink2
0: Tx: Transfer1
0: Rx: Blink1
0: Tx: Transfer2
0: Rx: Blink2
...
```

Related Issue
-----------
None


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
